web/styles/popovers.css

```diff
@@ -803,7 +803,8 @@ ul {
     }
 }
 
-#move_topic_modal {
+#move_topic_modal,
+#rename_topic_modal {
```

Comment:
Should this be changed to just `.message-move-form`?

Reply:
OK. Probably cleaning up the `new-style` is a thing I should try to figure out a plan for.